lib/auth0/api/authentication_endpoints.rb

```diff
@@ -215,12 +215,22 @@ def samlp_url(connection = UP_AUTH)
       # Return a WS-Federation URL.
       # @see https://auth0.com/docs/api/authentication#accept-request35
       # @param connection [string] Connection to use; empty to show all
+      # @param options [hash] Extra options; supports wtrealm, wctx, wreply
       # @return [url] WS-Federation URL
-      def wsfed_url(connection = UP_AUTH)
+      def wsfed_url(connection = UP_AUTH, options = {})
         request_params = {
-          whr: connection
+          whr: connection,
+          wtrealm: options[:wtrealm],
```

Comment:
Please add a test that asserts these properties are not being sent in the body when "null" or missing

Reply:
This just generates a URL, no HTTP requests